program-runtime/src/loaded_programs.rs

```diff
@@ -120,6 +126,20 @@ impl LoadedProgram {
             program: LoadedProgramType::BuiltIn(program),
         }
     }
+
```

Comment:
I don't think there is much use in being able to only recompile a loaded program, because on feature set changes, we also have to reverify them. So we might as well just create new `LoadedProgram` instances and always compile them when creating them.

Reply:
Yes, that makes sense. I'll move this block of code to `bpf_loader`